content/calico/stars_policy_demo/create_resources.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Create Resources"
+date: 2018-08-07T08:30:11-07:00
+weight: 1
+---
+
+Before creating network polices, let's create the required resources.
+
+Create a namespace called [stars](https://docs.projectcalico.org/v3.2/getting-started/kubernetes/tutorials/stars-policy/manifests/00-namespace.yaml). We will create frontend and backend services in this namespace in later steps
```

Comment:
I'd like to modify this experience to --> here is yaml for the namespace. During testing next week, we will be targeting these yaml files and pose the audience with a challenge question and force them to edit these files. Don't forget to link to k8s docs on how to create namespace

Reply:
will be pinging about this on Chime

Reply:
I think you forgot to clone the source repo before this command
cd ~/environment/calico_resources
kubectl apply -f 00-namespace.yaml

Reply:
Hi Peter, I have cloned the repo in content/prerequisites/clone.md, under commit 891580f5723b88649d5bd4e769ae57f3e5a88876 (https://github.com/aws-samples/eks-workshop/pull/111/commits/891580f5723b88649d5bd4e769ae57f3e5a88876)

Reply:
Could you move the cloning repo instructions into your chapter? this is mainly for keeping each chapter self-contained. You can add this right before creating namespace

Reply:
I have moved it to my chapter, and committed my changes: e910c9d661f7a51431701018a1a026a2b97d33f7